app/soundcloud.js

```diff
@@ -76,6 +77,10 @@ function onMediaStartedPlaying() {
   getTrackMetadata.call(this).then((trackMetadata) => {
     if (trackMetadata)
       this.emit('play', trackMetadata)
+      if (!compareTrackMetadata(this.trackMetadata, trackMetadata)) {
+        this.emit('new-track', trackMetadata)
```

Comment:
Hello @tersers and thank you for the contribution! Looks absolutely legit, I would only suggest a minor tweak.

The name of the new event above is a little misleading in my opinion, none of us in a few months will remember `new-track` stands for "a track that was not played previously started playing". What do you think about `play-new-track` or something slightly more descriptive? (It's hard to come up with something perfect but it would be nice to include the word "play").

Reply:
Thanks for the feedback! I'll take care of that tonight.

Reply:
Thanks for the feedback! I'll take care of that tonight. (I accidentally commented from my work account haha)